### PR TITLE
manual migrate R 4.3, libjpeg_turbo

### PR DESCRIPTION
**Pull request by mfansler**
Try migrating R 4.3.

Closes #18 

## Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mfansler** [comment]
@conda-forge-admin please rerender

**Comment by mfansler** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender
@conda-forge-admin please restart cis

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,16 +36,16 @@ requirements:
...
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
-    - {{ cdt('libselinux') }}  # [linux]
-    - {{ cdt('libxdamage') }}  # [linux]
-    - {{ cdt('libxxf86vm') }}  # [linux]
-    - {{ cdt('libxext') }}     # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:48]: these cdts should be regular conda packages

see https://github.com/conda-forge/qt-main-feedstock/pull/139/files

**Comment by h-vetinari** [comment]
Fixed in #18

**Pull request closed without merging**